charts/fluentd/templates/daemonset.yaml

```diff
@@ -31,6 +31,26 @@ spec:
               secretKeyRef:
                 name: {{ template "fullname" . }}
                 key: elasticsearch_port
+          - name:  FLUENT_ELASTICSEARCH_SCHEME
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "fullname" . }}
+                key: elasticsearch_scheme
+          {{ if .Values.elasticsearch.username }}
```

Comment:
This is to ensure that in cases where a username/password isn't required it doesn't result in an empty string being used, rather than username/password not being sent at all. This might not actually be necessary though - I'll check